README.md

```diff
@@ -12,33 +12,33 @@ Requirements
+| jenkins_agent_additional_args        | [deleteExistingClients, disableClientsUniqueId]                                                      | Additional arguments to send to the Swarm client jar                                              |
+| jenkins_swarm_client_version         | 3.3                                                                                                  | Version of the Swarm client to download                                                           |
+| jenkins_plugins_url                  | https://repo.jenkins-ci.org                                                                          | Base URL to download the client                                                                   |
+| jenkins_plugins_repo_path            | releases/org/jenkins-ci/plugins/swarm-client/{{ jenkins_swarm_client_version }}                      | URL between the base URL and the jar file name                                                    |
+| jenkins_swarm_client_jar             | swarm-client-{{ jenkins_swarm_client_version }}.jar                                                  | The file name for the swarm client jar                                                            |
+| jenkins_swarm_download_url           | {{ jenkins_plugins_url }}/{{ jenkins_plugins_repo_path }}/{{ jenkins_swarm_client_jar }}             | Full URL to the Swarm client jar                                                                  |
+| jenkins_swarm_path                   | /var/lib/jenkins                                                                                     | Path to the swarm client jar file                                                                 |
+| jenkins_swarm_config_path            | /etc/jenkins                                                                                         | For CentOS while it is using the init.d setup, this is where the swarm-client settings are stored |
+| jenkins_swarm_task_name              | Jenkins Swarm Client                                                                                 | Description for systemd                                                                           |
+| jenkins_swarm_systemd_path           | /lib/systemd/system                                                                                  | Path to systemd folder                                                                            |
+| jenkins_swarm_service_name           | swarm-client                                                                                         | Name of the systemd service                                                                       |
+| jenkins_swarm_client_wrapper_version | 2.0.3-pre                                                                                            | Windows Service Wrapper version                                                                   |
```

Comment:
default was updated to 2.0.3 and the jenkins_swarm_wrapper_download_url var was changed.